index.html

```diff
@@ -966,7 +966,8 @@ <h2><dfn>Coalesced events</dfn></h2>
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
             zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
             <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
-            that were coalesced into this event. For all other trusted event types, it is an empty list.
+            that were coalesced into this event, including a clone of the "parent" event itself.
```

Comment:
Minor nit: since the parent event can include slight interpolation / prediction it might be better to say something like including the event represented by the parent event?

Reply:
Yeah. I don't think the new statement is fully correct. The parent event might have different movementX/Y as @smaug---- also mentioned but it could also be resampled to what the user agent believes to be best representative of the current state (e.g. location, pressure and other similar stuff) of the pointer as @flackr mentioned. On the other hand the coalesced events are meant to be raw events received from the platfrom with no modifications on them.